python/pyspark/sql/connect/session.py

```diff
@@ -476,6 +476,11 @@ def createDataFrame(
 
     createDataFrame.__doc__ = PySparkSession.createDataFrame.__doc__
 
+    def _createCachedDataFrame(self, relationId: str) -> "DataFrame":
```

Comment:
this seems to be unused here?

Reply:
Removed. It will used in foreachBatch implementation (in follow up PRs)